armada/handlers/armada.py

```diff
@@ -82,11 +82,11 @@ def pre_flight_checks(self):
             if ct_type == 'git' and not git.check_available_repo(location):
                 raise ValueError(str("Invalid Url Path: " + location))
 
-            if not self.tiller.tiller_status():
```

Comment:
these checks are only running once before we start running any charts. we do not run them per chart.